packages/irmin-git/irmin-git.3.0.0/opam

```diff
@@ -32,7 +32,14 @@ depends: [
   "mtime"      {with-test & >= "1.0.0"}
   "alcotest"   {with-test}
 ]
-available: [ arch != "s390x" ] # temporary disable until ocaml-git works properly
+available: [
+  # temporary disable until ocaml-git works properly
+  arch != "s390x"
+
+  # Disabled on 32-bit platforms as the Dune build fails in CI:
+  & arch != "arm32"
```

Comment:
Is there any place where you may be calling Bytes.create with a size larger than 16Mb?

Reply:
> I would disable tests on those archs instead of marking it unavailable, and only make it unavailable at a later stage if an issue is reported

Done; thanks for the suggestion.

> Is there any place where you may be calling Bytes.create with a size larger than 16Mb?

The failure happens from within Dune, so I suspect it has to do with some peculiarities of `irmin-git`'s tests that make them difficult to build. Beyond that, I'm not sure :-)